front/app/containers/SiteMap/index.tsx

```diff
@@ -167,43 +146,38 @@ const SiteMap = () => {
     });
 
     return (
-      <Container>
+      <>
         <SiteMapMeta />
-        {!loaded && (
-          <Loading>
-            <Spinner />
-          </Loading>
-        )}
-        {loaded && (
```

Comment:
No use for loaded, based on projects being defined, since we don't even need projects in this component